TMessagesProj/src/main/java/org/telegram/ui/PassportActivity.java

```diff
@@ -1,3 +1,11 @@
+/*
+ * This is the source code of Telegram for Android v. 3.x.x
+ * It is licensed under GNU GPL v. 2 or later.
+ * You should have received a copy of the license in this archive (see LICENSE).
+ *
+ * Copyright Nikolai Kudashov, 2013-2017.
+ */
+
```

Comment:
This [notice](https://www.gnu.org/licenses/gpl-faq.html#NoticeInSourceFile) should put in every source file. But like [here](https://github.com/DrKLO/Telegram/blob/master/LICENSE#L293).